src/utils/backupKeyUtils.ts

```
@@ -99,7 +92,7 @@ export const detectBackupKeyFormat = (
       };
     }
   } catch (err) {
-    // Not JSON, might be an old format (just plain mnemonic)
+    errorService.reportError(err);
```

Not sure if we want to report the error as it can be caused by an old format 🤔

---

You’re right. I removed the try/catch to avoid the Sonar issue: “Handle this exception or don't catch it at all.”